bears/c_languages/CSecurityBear.py

```diff
@@ -0,0 +1,23 @@
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+
+
+@linter(executable="flawfinder", output_format="regex",
+        output_regex=r'.+:(?P<line>\d+):(?P<column>\d+):\s*'
+                     r'\[(?P<severity>\d)\]\s*'
```

Comment:
Is the square brackets around severity needed ? Isnt that redundant ?
Same with the curved brackets around origin